server/src/test/java/org/elasticsearch/cluster/coordination/RemoveSettingsCommandIT.java

```diff
@@ -54,11 +57,11 @@ public void testRemoveSettingsAbortedByUser() throws Exception {
                      ElasticsearchNodeCommand.ABORTED_BY_USER_MSG);
     }
 
+    @Test
     public void testRemoveSettingsSuccessful() throws Exception {
         internalCluster().setBootstrapMasterNodeIndex(0);
         String node = internalCluster().startNode();
-        client().admin().cluster().prepareUpdateSettings().setPersistentSettings(Settings.builder()
-                                                                                     .put(DiskThresholdSettings.CLUSTER_ROUTING_ALLOCATION_DISK_THRESHOLD_ENABLED_SETTING.getKey(), false).build()).get();
+        execute("set global persistent cluster.routing.allocation.disk.threshold_enabled = false");
```

Comment:
It seems cleaner like that, but maybe makes future changes on settings a bit harder. I'm not insisting though, as it would only be a single find or find/replace.